### DISCUSSION TITLE
What does "Filters:" In RelatedListLayout stands for?

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForceCRM#17146] [General] [answered]
Common sense telling me that "Filters" in RelatedListLayout panel should filter records as per given filter but this doesnt work.
Am I missing something?

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/50770738/31440f3d-47f7-45af-9219-aecad35b63d0)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
![Screenshot_20230619_215116_Chrome~2](https://github.com/YetiForceCompany/YetiForceCRM/assets/10198654/69794d29-84fc-4729-be0e-bec7fbcb953f)

**Reply by Marshal4582** [reply]
Blind me
